static/app/components/eventOrGroupTitle.spec.jsx

```diff
@@ -80,4 +81,18 @@ describe('EventOrGroupTitle', function () {
 
     expect(screen.getByText('metadata title')).toBeInTheDocument();
   });
+
+  it('does not render stack trace when issueCategory is performance', () => {
+    render(
+      <EventOrGroupTitle
+        data={{
+          ...data,
+          issueCategory: IssueCategory.PERFORMANCE,
+        }}
+        withStackTracePreview
+      />
+    );
+
+    expect(screen.queryByTestId('stacktrace-preview')).not.toBeInTheDocument();
```

Comment:
do we need to mouse over it or anything?

Reply:
This test ID is on the container div that captures the mouse events so it should always be rendered